WooCommerce/Classes/ViewRelated/Products/ShareProduct/ShareProductCoordinator.swift

```diff
@@ -16,54 +16,54 @@ final class ShareProductCoordinator: Coordinator {
 
     private var bottomSheetPresenter: BottomSheetPresenter?
 
-    private init(site: Site?,
+    private init(siteID: Int64,
                  productURL: URL,
                  productName: String,
                  shareSheetAnchorView: UIView?,
                  shareSheetAnchorItem: UIBarButtonItem?,
-                 featureFlagService: FeatureFlagService,
+                 shareProductEligibilityChecker: ShareProductAIEligibilityChecker,
```

Comment:
super nit: since this coordinator is about "share product", it can probably be skipped here
```suggestion
                 eligibilityChecker: ShareProductAIEligibilityChecker,
```

Reply:
Good point! Updated in 7118deba2da16bddac818241b8b9bd22b8187177.